pages/settings/settings.js

```diff
@@ -347,7 +347,18 @@ function onKeyMapChange (e) {
       keyMapSettings = {}
     }
 
+    var currentKeyMap = {...defaultKeyMap, ...keyMapSettings}
+
     keyMapSettings[action] = parseKeyInput(newValue)
+
+    // if the keymap before change already contains this key combination...
+    Object.keys(currentKeyMap).forEach(keySetting => {
+      if (currentKeyMap[keySetting] === keyMapSettings[action]) {
```

Comment:
Another thing to consider here is that it's possible to have multiple shortcuts for a single action separated by commas; you probably need to split the shortcut value into an array of items, check if any of the items in the array are identical to the new shortcut, and remove them from the array if they are.